README.md

```diff
@@ -1,3 +1,51 @@
-# Cocos Creator 3D
-
-TO BE DONE
+# Cocos Creator 3D
+
+Cocos Creator 3D is a complete package of game development tools and workflow, including a game engine, resource management, scene editing, game preview, debug and publish one project to multiple platforms. Based on Cocos Creator,it has been aggresively refactored to meet the complex needs of 3D projects while retaining most of the original features where applicable.
+
+This is the repo for Cocos Creator 3D engine, which is written mostly in TypeScript and conforms to the HTML5 standard, supporting mainstream desktop and mobile browsers and all sorts of WebGL-interfaced native in-app platforms. The engine itself is mostly self-contained, with full-fledged runtime modules including lighting, material, particle, animation, physical, UI, terrain, sound, resource and scene-graph management, etc.

```

Comment:
This repo is the engine part of Cocos Creator 3D, it's mainly written in TypeScript and support users to use TypeScript or ES6 to write game logics.  The engine itself is mostly self-contained, with full-fledged runtime modules including lighting, material, particle, animation, physical, UI, terrain, sound, resource and scene-graph management, etc. It supports both native and web platforms, including Windows, Mac, iOS, Android, Web. What's more exciting is that it supports rapidly expanding instant gaming platforms like WeChat Mini Game and Facebook Instant Games.